test/steps/operator.go

```diff
@@ -61,32 +61,30 @@ func (data *Data) kogitoOperatorIsDeployedWithDependencies(dependencies string)
 		}
 	}
 
+	// Dependent operators
 	operatorSource := framework.CommunityCatalog
 	if !framework.IsOpenshift() {
 		operatorSource = framework.OperatorHubCatalog
 	}
 
-	// Install operator dependencies
-	var installedOperators []string
+	// Install and wait for operator dependencies
+	// Do it one by one due to racing condition in OLM (https://github.com/operator-framework/operator-lifecycle-manager/issues/1704)
 	for _, dependentOperator := range framework.KogitoOperatorDependencies {
 		if strings.Contains(dependencies, dependentOperator) {
 			if err := framework.InstallKogitoOperatorDependency(data.Namespace, dependentOperator, operatorSource); err != nil {
 				return err
 			}
-			installedOperators = append(installedOperators, dependentOperator)
+			if err := framework.WaitForKogitoOperatorDependencyRunning(data.Namespace, dependentOperator, operatorSource); err != nil {
+				return err
+			}
 		}
 	}
 
-	// Wait for kogito operator to be running
+	// Wait for Kogito operator running
+	// This is put at the end because there should not be any racing condition as install is done via yaml and not OLM
```

Comment:
@Sgitario I let this one at the end as there should not be any conflict. no need to wait first for Kogito Operator running